### ISSUE TITLE
Error message after update of Home Assistant

### ISSUE THREAD

**Issue by countrr** [pnbruckner/ha-illuminance#43]
After updating Home Assistant to:
_Home Assistant 2023.4.1
Supervisor 2023.04.0
Operating System 9.5
Frontend 20230406.1 - latest_
I get the following error, which prevents HA to restart:
_Platform error sensor.illuminance - No module named 'homeassistant.components.darksky'_
Commenting out these lines in configuration.yaml solves the issue, of course disabling the sensor :( :
```
#  - platform: illuminance
#    entity_id: sensor.openweathermap_condition
```
Any hint on how to resolve this? Thanks!

**Comment by Ragr3n** [comment]
Looks like Dark Sky has been removed from Home Assistant. 

From latest release notes:
Dark Sky has been removed. Apple acquired Dark Sky, and the API has now been shut down. ([@gjohansson-ST](https://github.com/gjohansson-ST) - [#90322](https://github.com/home-assistant/core/pull/90322))

Guessing the best way to go is to use one of the other supported platforms.

**Comment by countrr** [comment]
> Guessing the best way to go is to use one of the other supported platforms.

I wasn't using that platform, that is the strange thing. In my code above I was calling this platform:
`entity_id: sensor.openweathermap_condition`

**Comment by pnbruckner** [comment]
@countrr I already fixed this problem with release 4.0.1. Reinstalling this custom integration upgraded you to the new version, which resolves the issue.

**Comment by countrr** [comment]
thanks!
